subgraph/src/fleek-nfa.ts

```diff
@@ -343,7 +425,7 @@ export function handleTransfer(event: TransferEvent): void {
   accessPointEntity.score = BigInt.fromU32(0);
   accessPointEntity.contentVerified = false;
   accessPointEntity.nameVerified = false;
-  accessPointEntity.status = 'DRAFT'; // Since a `ChangeAccessPointCreationStatus` event is emitted instantly after `NewAccessPoint`, the status will be updated in that handler.
+  accessPointEntity.creationStatus = 'DRAFT'; // Since a `ChangeAccessPointCreationStatus` event is emitted instantly after `NewAccessPoint`, the status will be updated in that handler.
```

Comment:
I think it would be cool have a enum for this also
```ts
enum CreationStatus {
  Draft = 'DRAFT',
  Approved = 'APPROVED',
  ...
}
```

Reply:
Discussed on Discord with @zoruka 

Summary:

The code does not compile with the use of this enum. Zoruka thinks the compiler fails in converting the enum to its plain js counterpart.

Not going to be implemented.

The error:

```
ERROR TS2322: Type 'i32' is not assignable to type '~lib/string/String'.

         accessPointEntity.creationStatus = CreationStatus.Draft;
```